Guide.docc/MigrationGuide.md

```diff
@@ -43,8 +43,7 @@ Here you will find articles and code examples that will:
 > Important: The Swift 6 language mode is _opt-in_.
 Existing projects will not switch to this mode without configuration changes.
 >
-> There is an important distinction between the _compiler version_
-and _language mode_.
+> There is a distinction between the _compiler version_ and _language mode_.
```

Comment:
This is within a note marked "Important". I don't think restating that this is also important is adding anything.